boards.txt

```diff
@@ -228,7 +228,11 @@ Generic_nRF51822.build.board=GENERIC
 Generic_nRF51822.build.core=nRF5
 Generic_nRF51822.build.variant=Generic
 Generic_nRF51822.build.variant_system_lib=
+<<<<<<< Updated upstream
+Generic_nRF51822.build.extra_flags=-DNRF51 -DNRF51822
+=======
 Generic_nRF51822.build.extra_flags=-DNRF51
+>>>>>>> Stashed changes
```

Comment:
@carlosperate Do you have some advice on what to do on the issue with the pins that aren't broken out? For now I'm leaving them with a big value like: 
`const uint32_t g_ADigitalPinMap[] = {
  0,
  1,
  2,
  3,
  50}`

Is that ok, as I can't use -1 or 0 ?